cockroachdb/tests/docker/docker-compose.yaml

```diff
@@ -12,6 +12,8 @@ services:
     ports:
       - "26257:26257"
       - "8080:8080"
+#    volumes:  # Uncomment this section to mount the sql script. See /tests/README.md.
+#      - ./sql.sh:/sql.sh
```

Comment:
Is there any harm in leaving this uncommented? I think that the less that we rely on commenting / uncommenting the better.

Reply:
Not fanny but I think it's ok to leave it, I agree with your comment